Server/modules/webdownloader.py

```diff
@@ -0,0 +1,31 @@
+class STModule:
+    def __init__(self):
+        self.name = 'webdownloader'
+        self.description = 'Download a file to a destination path.'
+        self.author = '@b2az'
+        self.options = {
+            'URL': {
+                'Description': 'The URL of the file.',
+                'Required': True,
+                'Value': "https://cdn.instructables.com/F23/YMO0/FPIUQOJF/F23YMO0FPIUQOJF.LARGE.jpg"
```

Comment:
Can you get rid of the default value? 

```suggestion
                'Value': ""
```
I'll definitely merge it once that's taken care of.